src/app/modules/comment.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable linebreak-style */
+import { overlay, popUp, closePopUpHandler } from './pop.js';
+import apiCall, { appID, error } from './utilities.js';
+
+const sendComment = async (e, beerInfo) => {
+  try {
+    e.preventDefault();
+    const form = document.querySelector('#addPost');
+    const itemId = beerInfo.id;
+    const username = form.children[0].value;
+    const comment = form.children[1].textContent;
+    const newComment = {
+      item_id: itemId,
+      username,
+      comment,
+    };
+    const response = await apiCall(`${appID}/comments`, 'POST', newComment, true);
```

Comment:
your not using the response variable so instead of using that you can only write await apicall...